index.js

```diff
@@ -1,26 +1,63 @@
-const core = require('@actions/core')
-const Jira = require('./common/net/Jira')
+const core = require('@actions/core');
+const Jira = require('./common/net/Jira');
 const github = require('@actions/github');
 
-const githubToken = process.env.GITHUB_TOKEN
-const octokit = github.getOctokit(githubToken);
```

Comment:
Only for noticed!) 
It didn't work correct without this strings when i wrote it) All is fine if you tested it)
const githubToken = process.env.GITHUB_TOKEN
const octokit = github.getOctokit(githubToken);